kie-based-services/pom.xml

```diff
@@ -415,6 +415,14 @@
 					<groupId>org.yaml</groupId>
 					<artifactId>snakeyaml</artifactId>
 				</exclusion>
+				<exclusion>
+					<groupId>org.apache.tomcat.embed</groupId>
+    				<artifactId>tomcat-embed-core</artifactId>

```

Comment:
The indentation on both of these `artifactId` elements is wrong.

Reply:
Why are the exclusions needed? Is there no new version of `spring-boot-starter-web` that includes the new tomcat artifacts?

Reply:
The updated `spring-boot-starter-web` v 2.7.17 already includes these patched artifacts.